demos/python_demos/rag_chatbot/README.md

```diff
@@ -0,0 +1,163 @@
+
+```bash
+INFO:nncf:NNCF initialized successfully. Supported frameworks detected: torch, onnx, openvino
+usage: compress_model.py [-h] --model
+                         {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+
+Script to compress LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+                        Select the LLM model out of supported list
+
+python compress_model.py --model ${SELECTED_MODEL}
+
+
+```
+Running this script will create new directories with compressed versions of the model with FP16, INT8 and INT4 precisions.
+The compressed models can be used in place of the original as they have compatible inputs and outputs.
+
+```bash
+ls  -1 | grep tiny-llama-1b-chat
+tiny-llama-1b-chat
+tiny-llama-1b-chat_FP16
+tiny-llama-1b-chat_INT4_compressed_weights
+tiny-llama-1b-chat_INT8_compressed_weights
+```
+
+> **NOTE** Applying quantization to model weights may impact the model accuracy. Please test and verify that the results are of acceptable quality for your use case.
+
+> **NOTE** On target devices that natively support FP16 precision (i.e. GPU), OpenVINO automatically adjusts the precision from FP32 to FP16. This improves the performance and typically does not impact accuracy. Original precision can be enforced with `ov_config` key:
+`{"INFERENCE_PRECISION_HINT": "f32"}`.
+
+## Download embedding model
+
+Download the model using `download_embedding_model.py` script:
+
+```bash
+python download_embedding_model.py --help
+usage: download_embedding_model.py [-h] --model {all-mpnet-base-v2,text2vec-large-chinese}
+
+Script to download LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model {all-mpnet-base-v2,text2vec-large-chinese}
+                        Select the LLM model out of supported list
+
+python download_embedding_model.py --model all-mpnet-base-v2
+
+```
+The model will appear in `./all-mpnet-base-v2` directory.
+
+## Download test documents for knowledge base
+We will use single CSV file with [FIFA World Cups information](https://raw.githubusercontent.com/rangelak/fifa-stats/master/data/fifa-world-cup.csv).
+```bash
+wget -P documents/ https://raw.githubusercontent.com/rangelak/fifa-stats/master/data/fifa-world-cup.csv
+```
+
```

Comment:
Maybe we could have a summary of the main catalog here? So the user can confirm everything is in place before deployment.

Reply:
Changed to aipc.txt as suggested by Ryan.